Sources/SmileID/Classes/Networking/Models/SmartSelfieRequest.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+struct SmartSelfieRequestImage: Encodable {
+    let key: String
+    let filename: String
+    let data: Data
+    let mimeType: String
+
+    init?(withImage image: Data, forKey key: String, forName name: String) {
+        self.key = key
+        mimeType = "image/jpeg"
+        filename = name
+        data = image
+    }
+}
+
+public struct SmartSelfieRequest: Encodable {
```

Comment:
instead of creating a struct for this, can you encode them as top level parameters in the network interface, since they are each in independent form data part? This will also allow some parameters to be explicit where required. For example, userId is not allowed to be nil for SmartSelfieAuth, but here it is marked as optional

see here: https://github.com/smileidentity/android/blob/b7bade606af4630b6b58b8a76d34aafa69ab509b/lib/src/main/java/com/smileidentity/networking/SmileIDService.kt#L82-L90